testing/ufw-extras/APKBUILD

```diff
@@ -0,0 +1,21 @@
+# Contributor: TBK <alpine@jjtc.eu>
+# Maintainer:
+pkgname=ufw-extras
+pkgver=0.5
+pkgrel=0
+pkgdesc="Extra configuration files for uncomplicated firewall (ufw)."
+url="https://github.com/xyproto/ufw-extras"
+arch="noarch"
+license="GPL-2.0-or-later"
+options="!check" # No test suite
+depends="ufw"
+source="$pkgname-$pkgver.tar.gz::https://github.com/xyproto/ufw-extras/archive/$pkgver.tar.gz"
+
+package() {
+	for f in "ufw-"*
+	do
+		install -Dm644 "$f" "$pkgdir/etc/ufw/applications.d/$(basename $f)"
```

Comment:
is the basename necessary ? the for loop is not transversing a directory.

also i think the more common usage of the for loop is with `; do` instead of the `do` in the next line

Reply:
otherwise LGTM

Reply:
Thx for the review. I have changed the for loop syntax and removed basename.